src/editors/select.js

```diff
@@ -318,4 +318,4 @@ JSONEditor.defaults.editors.select = JSONEditor.AbstractEditor.extend({
 
     this._super();
   }
-});
+});
```

Comment:
Can you revert this file?  It's just a whitespace change and I want to keep the pull request as small as possible to make it easier to merge.